prediction_market_agent_tooling/markets/manifold/manifold.py

```
@@ -110,8 +111,8 @@ def get_markets(
         filter_by: FilterBy = FilterBy.OPEN,
         created_after: t.Optional[DatetimeUTC] = None,
         excluded_questions: set[str] | None = None,
-        fetch_categorical_markets: bool = False,
-        fetch_scalar_markets: bool = False,
+        market_types: list[MarketType] = [MarketType.ALL],
```

Ruff (0.11.9)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)